### DISCUSSION TITLE
方便的flomo chrome extension，一键保存到flomo[已上架store]

### DISCUSSION THREAD

**Discussion by YIREN1** [flomoapp/3rd-party-tools#12] [分享创造]
https://github.com/YIREN1/chrome-extension-flomo-api

## 用法

**已发布 chrome web store， 久等了**
https://chrome.google.com/webstore/detail/flomo-api/bliaamgfpeogldcelkmkegfofapaocfn/related?hl=en&authuser=0

1. 谷歌应用商店下载Flomo API

2. 在“添加至chrome”之前打开的网页需要刷新一下，才能使用插件

3. 第一次打开插件,需要在popup（点击插件图标弹出）填写一下flomo[申请到的api接口地址](https://flomoapp.com/mine?source=incoming_webhook).保存后即可使用.
![Screen Shot 2020-12-20 at 20 40 27](https://user-images.githubusercontent.com/39056160/102713638-a9a49d80-4304-11eb-88c9-ebe659d6fd5e.png)
4. 选中文字，点击弹出的flomo 图标， 即可保存到flomo
![Screen Shot 2020-12-20 at 20 49 24](https://user-images.githubusercontent.com/39056160/102713687-0607bd00-4305-11eb-98fb-0fc7dec181fa.png)
5. **如果您不喜欢弹出按钮的方式，可以关掉'extension enabled'开关，此举将禁止在所有网站弹出按钮，仍可以通过右键选择'Save Text to Flomo'来保存到flomo**

- [详细用法](https://github.com/YIREN1/chrome-extension-flomo-api/blob/main/usage_zh.md)
- 12.31 v0.2.3 update
  - 增加了右键contextmenus 的备用方案
  - 增加了默认标签，#chrome 和 from url

- **本着简化记录流程（只需点击一下）的理念，用业余时间写了这个extension，希望能符合flomo“记得更多”的初衷, have fun 🎉**

**Comment by lightory** [comment]
👍

**Comment by lightory** [comment]
https://github.com/YIREN1/chrome-extension-flomo-api
这个链接打不开，Page not found

**Reply by YIREN1** [reply]
不好意思忘记设成public了，等1min

**Reply by YIREN1** [reply]
应该可以了

**Comment by zhxliao** [comment]
坐等 store 版

**Comment by lightory** [comment]
上架 Store 了不？

**Reply by YIREN1** [reply]
我刚看了一眼还没有😂现在可能store查的严了。。前几天他退回来我改了一次，现在还没回复
![Screen Shot 2020-12-26 at 08 05 17](https://user-images.githubusercontent.com/39056160/103143451-2beff000-4751-11eb-9ddd-390d560233cf.png)

**Reply by YIREN1** [reply]
我猜他们放圣诞节呢。。

**Reply by YIREN1** [reply]
https://chrome.google.com/webstore/detail/flomo-api/bliaamgfpeogldcelkmkegfofapaocfn/related?hl=en&authuser=0
上架了😂

**Comment by YT-er** [comment]
好像无法和google翻译同时使用，只会显示google翻译的小图标

**Reply by YIREN1** [reply]
可能和别的相同性质的插件会冲突，我研究一下有没有解决办法

**Comment by whistlesun** [comment]
有个问题，能把html标签去掉吗，存上来的还得需要二次编辑。如下图

![Jietu20210114-204949](https://user-images.githubusercontent.com/1300426/104593060-1748ad00-56aa-11eb-8016-61e0486c566c.jpg)

**Reply by YIREN1** [reply]
oh no,这是个问题。。感谢提醒，我看一眼

**Comment by gaoyajiang** [comment]
https://support.qq.com/products/297045/link-jump?jump=https%3A%2F%2Fflomoapp.com%2Fmine%3Fsource%3Dincoming_webhook
这个链接打不开~

**Reply by YIREN1** [reply]
更新了一下，您可以试试
